gammapy/data/pointing.py

```diff
@@ -0,0 +1,135 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+from astropy.utils import lazyproperty
+from astropy.units import Quantity
+from astropy.table import Table
+from astropy.coordinates import SkyCoord, AltAz
+from ..utils.scripts import make_path
+from ..time.utils import time_ref_from_dict
+from .utils import _earth_location_from_dict
+
+__all__ = [
+    'PointingInfo',
+]
+
+
+class PointingInfo(object):
+    """IACT array pointing info.
+
+    TODO: link to open spec.
+    TODO: share code with the `~gammapy.data.EventList` and `~gammapy.data.ObservationTable` classes.
+
+    This class has many cached properties.
+    Should be used as read-only.
+
+    Parameters
+    ----------
+    table : `~astropy.table.Table`
+        Table (with meta header info) on pointing
+
+    Examples
+    --------
+
+    >>> from gammapy.data import PointingInfo
+    >>> pointing_info = PointingInfo.read('$GAMMAPY_EXTRA/test_datasets/hess_event_list.fits')
+    >>> print(pointing_info)
```

Comment:
Your traceback shows that you're running with Astropy 1.0.4
The `fits` format for Astropy `Time` was added in Astropy 1.1.

Please update your Astropy to 1.1 or even better 1.2.